### DISCUSSION TITLE
Tititopher68's firmware

### DISCUSSION THREAD

**Discussion by kted** [Jyers/Marlin#2075] [General Discussion]
Hey guys, I was using tititopher68's firmware after jyers stopped development, up until 3 months ago, when I switched to klipper.

I deleted the bookmark to his discord channel, thinking I won't be needing it any more.

Now I do need it for a newbie friend who just got an Ender 3 V2.

I still have an older version I can compile, but I want to see what's evolved after I left.
Does anyone have the link to his firmware channel, the french-speaking one?

**Comment by darco415** [comment]
He got booted from the French discord channel...I have his latest...but I did go back to Klipper

**Reply by RV-from-be** [reply]
@darco415 
> He got booted from the French discord channel...I have his latest...but I did go back to Klipper

Why are you talking nonsense! He didn't get booted. He just decided to leave the French Discord. The administrators would even like him to come back. It's just his choice....because of "fat jerks fan of a Peruvian" like the ones that plagued this Github and ended up pissing off Jyers, Tititopher68 and others!

**Reply by darco415** [reply]
If he decided to leave, why didnt he inform us of his plans  in the discord channel?  He had just posted another "final" fw build and POOF he was gone. The channel admin posted an announcement he was no longer active in channel but nothing directly from him.

**Reply by Nicog77** [reply]
Now he's back.

**Reply by kted** [reply]
Do you have an invite to his channel?

**Reply by Nicog77** [reply]
https://discord.gg/NS4zYd23dX
